p4include/psa.p4

```diff
@@ -486,25 +579,15 @@ extern ActionProfile {
   }
   */
 }
+// END:ActionProfile_extern
```

Comment:
Why not have the comments for these?

Reply:
Sorry, not sure which things you mean by "these".

Reply:
ActionProfiles, selectors.

Reply:
the comments were deleted

Reply:
I can't even find in the git commit history of the p4lang/p4-spec repository a version of psa.p4 that had these comments in there.  Not sure when they were removed.  That text appears to be completely within the PSA specification now, in the section describing the ActionProfile extern.